specification/evaluation-context/evaluation-context.md

```diff
@@ -1,5 +1,41 @@
 # Evaluation Context
 
+**Status**: [Experimental](../README.md#document-statuses)
+
 ## Overview
 
-// TODO: this is a stub.
+The `evaluation context` provides ambient information for the purposes of flag evaluation. Contextual data may be used as the basis for targeting, including rule-based evaluation, overrides for specific subjects, or fractional flag evaluation.
+
+### Fields
+
+##### Requirement 3.1
+
+> The `evaluation context` structure **MUST** define a required `targeting key` field of type string, identifying the subject of the flag evaluation.
+
+The targeting key uniquely identifies the subject (end-user, or client service) of a flag evaluation. Providers may use this as a unique identifier for fractional-flags, rules, or overrides targeting specific users.
+
+##### Requirement 3.2
+
+> The evaluation context **MUST** define the following optional fields: `email` (string), `first name` (string), `last name`(string), `name`(string), `client ip`(string), `tz`(string), `locale`(string), `date`(string | date), `environment`(string), `application`(string), and `version`(string).
```

Comment:
Tables look awful in the normative statements/JSON, for now I'm mentioning the fields in the statements and leaving the table in the auxiliary non-normative section.

Reply:
>`first name` (string), `last name`(string)

Not all cultures have a firstname/lastname construct. I also don't see what advantage you get for flag evaluation by having first and last names separately over just the `name` field.

Reply:
I see your point @agentgonzo , but `firstName` and `lastName` are defined properties in [LaunchDarkly's context](https://docs.launchdarkly.com/home/users/attributes), so not having them might add some unpredictability between that provider and others. They are also included in the [standard OIDC claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims).

Let's see what others think.

Reply:
I've removed `firstName`, `lastName` and `name`.